### PR TITLE
Hplu/fix linting

### PR DESCRIPTION
**Pull request by haipinglu**
Fixes linting errors and updates configs and renames tests with five commits.

### Description
- Update configuration settings (to only three files) and rename test to tests
- Black
  - Run `black` to autoformat everything with updated settings. 51 files reformatted, 13 files left unchanged.
  - Before: 300+ flake8 errors. After: 68 flake8 errors (typo in commit description)
- Manual 1
  - Manually fix the remaining 68 errors. 
  - Remaining (after Black) overlength errors are ignored via configuration (all in docstrings), **not individually!!!**
  - 17 files have been changes in total.
  - Not to ignore tests. Instead, we should write more complete test cases (rather than just import) in future.
- Manual 2
  - Problems pointed out by Xianyuan in PR#40 have been fixed, with three files changed.
- Linting and pre-commit have been switched on. Hopefully it can pass but if not, it should be fixable.
### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).

### CODE REVIEW

**Comment by haipinglu** [comment]
Linting passed. I have set to ignore those print checks by https://github.com/jbkahn/flake8-print because I do not understand the purpose and cannot fix it.
Pre-commit also found some file encoding inconsistence, which can be fixed later.

**Comment by haipinglu** [comment]
Not sure why pre-commit black failed due to one file not matching the standard (but OK on my WIN machine and HPC). I do not have ubuntu-latest (18.04) so I cannot figure out what needs to be changed to digits_access.py. This is frustrating if we want to do pre-commit. Otherwise, we may just use black without pre-commit check.

black....................................................................Failed
- hook id: black
- files were modified by this hook

reformatted /home/runner/work/pykale/pykale/kale/loaddata/digits_access.py
All done! ✨ 🍰 ✨
1 file reformatted, 39 files left unchanged.

**Comment by haipinglu** [comment]
Pre-commit black works now, finally!

**Review by pz-white — APPROVED** [review]
GraphML part looks good for me, the 120 max length limit is the better reformat now. Thanks!

**Review by XianyuanLiu — APPROVED** [review]
Reviewed the computer vision codes. Pretty good!

**Review by RaivoKoot — APPROVED** [review]

File: examples/cifar_cnntransformer/main.py [hunk 1]
```diff
@@ -8,7 +8,8 @@
 import argparse
 import warnings
 import sys
-sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '../..')))
+
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), "../..")))
```
[the remark this answers is not in the document]
Comment (RaivoKoot) [reply examples/cifar_cnntransformer/main.py:13]: I see that this I sone of our cards in TODO list. To discuss later.

**Review by sz144 — APPROVED** [review]
I have reviewed the changes in MPCA part, which works for me.

File: examples/cifar_cnntransformer/main.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (haipinglu) [reply examples/cifar_cnntransformer/main.py:13]: Yes, https://github.com/pykale/pykale/projects/9#card-52914703

**Pull request merged**